app/components/Pagination.tsx

```diff
@@ -0,0 +1,67 @@
+import { ChevronLeftIcon, ChevronRightIcon } from "@chakra-ui/icons";
+import { Box, Flex, HStack } from "@nycplanning/streetscape";
+import { useSearchParams } from "@remix-run/react";
+import { Button } from "@chakra-ui/button";
+import { Link } from "@remix-run/react";
+import { IconButton } from "@chakra-ui/react";
+
+export interface PaginationProps{
+    total: number;
+    path: string,
+};
+
+export const Pagination = ({total, path}: PaginationProps) => {
+    // where do we want to limit to only having 7 as limit
+    const [searchParams, setSearchParams] = useSearchParams();
+    const limit = 7; // Number(searchParams.get("limit"));
+    const offset = Number(searchParams.get("offset"));
+
+    const currentPage = offset === 0 ? 1 : offset / limit + 1;
```

Comment:
Would this also give `1` in the case where offset is `0`? If so, it seems more succinct to me.

```suggestion
    const currentPage = offset / limit + 1;
```